marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -127,7 +127,7 @@ function buildAllMetadata(parentPropertyName, entityModel, entityName) {
       propertyMetadata["properties"] = metaData["allPropertiesMetadata"];
       propertyMetadataObject["properties"] = metaData["allPropertiesMetadata"];
 
-      granularPropertyMetadata = {...granularPropertyMetadata, ...metaData["granularPropertyMetadata"]};
+      granularPropertyMetadata = Object.assign({},granularPropertyMetadata, metaData["granularPropertyMetadata"]);
```

Comment:
Unfortunately, the spread syntax isn't supported in ML 9.

Reply:
In 'manageCustomStep.sjs' , I used spread operator. I believe that should fail as well

Reply:
updated there now as well.